local.mk

```diff
@@ -1,5 +1,5 @@
 # Directly use the submodule source code for builds
-INGENIC_VIDEOCAP_OVERRIDE_SRCDIR=/src/submodules/ingenic_videocap
+# INGENIC_VIDEOCAP_OVERRIDE_SRCDIR=/src/submodules/ingenic_videocap
```

Comment:
Why do you comment this out here?

Reply:
I don't remember actually. I can revert this no problem.